### PR TITLE
#167503053 Setup Review Apps on Heroku

### PR DESCRIPTION
**Pull request by Bastieno**
#### What does this PR do?
This pull request sets up Review apps on Heroku

#### Description of Task to be completed
- generate app.json file on Heroku
- add app.json file

#### How should this be manually tested?
[vali-1kbideas-staging](https://vali-1kbideas-staging.herokuapp.com)
[vali-1kbideas](https://vali-1kbideas.herokuapp.com)

#### Any Background Context You want to provide?
This pull request needs to be merged to the develop branch as develop is the default branch to enable Review app to work.

#### What are the relevant pivotal tracker stories?

[#167503053](https://pivotaltracker.com/story/show/167503053)

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "addons": [
+      "heroku-postgresql"
+    ],
+    "buildpacks": [
+      {
+        "url": "heroku/nodejs"
+      }
+    ],
+    "stack": "heroku-18"
+  }
```
Comment (Merkll) [inline app.json]: Add new line

**Review by Merkll — APPROVED** [review]
Good job

**Review by ibkadeeko — APPROVED** [review]

**Review by Nennyfills — APPROVED** [review]
Good background context made a lot of sense!

File: app.json (hunk 1 above)
Comment (Nennyfills) [reply app.json]: Is not a new line issue, the closing tag is not properly indented.

File: app.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+    "name": "vali-ah-backend",
+    "description": "1kbIdeas: The one-stop shop for all the self-help ideas you need",
+    "scripts": {},
+    "env": {
+      "NODE_ENV": {
```
Comment (iverenshaguy) [inline app.json]: DATABASE_URL and PORT are also required or don't they need to be included?

File: app.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{
+    "name": "vali-ah-backend",
+    "description": "1kbIdeas: The one-stop shop for all the self-help ideas you need",
+    "scripts": {},
```
Comment (iverenshaguy) [inline app.json]: Please include a postdeploy script to help set up the database after each deploy. e.g 
```suggestion
    "scripts": {
      "postdeploy": "npm run db:migrate"
    },
```

**Review by iverenshaguy — CHANGES_REQUESTED** [review]
Good job! Please review my comments

File: app.json (hunk 2 above)
Comment (Bastieno) [reply app.json]: I really do not know. Can you please provide more insight on that?

**Review by iverenshaguy — APPROVED** [review]

**Pull request merged**